scripts/policy/protocols/ssl/decryption.zeek

```diff
@@ -0,0 +1,106 @@
+#! Decrypt SSL/TLS payloads
+
+@load base/frameworks/input
+@load base/frameworks/notice
+@load base/protocols/conn
+@load base/protocols/ssl
+
+module SSL;
+
+# Local
+const input_stream_name = "input-tls-keylog-file";
```

Comment:
Is there a reason to have this as a const?

Reply:
Not really, I just didn't see a use-case for making this &redef.